### ISSUE TITLE
1-click "Tabs aside"

### ISSUE THREAD

**Issue by irvnriir** [tim-we/tabs-aside#83]
LMB click on the toolbar item should { Save And Close / Close (if saved) } the tabs, while the current menu there -- should be in the Context Menu (before/after the current Sessions list) .

**Comment by tim-we** [comment]
I don't think this should be the default behavior (new users, accidental click, ...) but I could see this being an option that can be enabled.

**Comment by irvnriir** [comment]
its completely fine if its optional, just p.s.: if it works smooth enough, thats not an issues (it is not like deleting something) .

**Comment by tim-we** [comment]
The other thing is that the browser action context menu is limited to [6 entries](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/menus/ACTION_MENU_TOP_LEVEL_LIMIT), that would mean moving the quick restore menu items into a submenu.

**Comment by irvnriir** [comment]
isn't it already low anyway ?

**Comment by kabaur** [comment]
I 2nd irvnriir thought, that the Tabs aside entry should be the 1-click default.
 
I used OneTab before where this is the default beahviour. There the entry limitation is solved by shifting every menu item in a single ("root") submenu. So you right click the icon > hover the single submenu entry > and then have 10 entries with one of them beeing a further submenu. Its less then a second for the first submenu to open and since its just were the mouse is anyway it doesn't bother (me) at all. It would be cool to have that in tabs-aside.

PS: biggest downside of OneTab is no sync and no active sessions

**Comment by tim-we** [comment]
If I find the time for it I will add an option for this. But unfortunately I do not have a lot of time for Tabs Aside right now so that might take a while.
